src/v2/Apps/Partners/Components/PartnersFilteredCells.tsx

```diff
@@ -0,0 +1,172 @@
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import {
+  PartnerCellFragmentContainer,
+  PartnerCellPlaceholder,
+} from "v2/Components/Cells/PartnerCell"
+import { PartnersFilteredCells_viewer } from "v2/__generated__/PartnersFilteredCells_viewer.graphql"
+import { PartnersFilteredCellsQuery } from "v2/__generated__/PartnersFilteredCellsQuery.graphql"
+import { useSystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import {
+  Box,
+  Button,
+  Column,
+  GridColumns,
+  Skeleton,
+  SkeletonText,
+  Text,
+} from "@artsy/palette"
+import { compact } from "lodash"
+
+interface PartnersFilteredCellsProps {
+  viewer: PartnersFilteredCells_viewer
+}
+
+const PartnersFilteredCells: FC<PartnersFilteredCellsProps> = ({ viewer }) => {
+  const response = viewer.filterPartners
+
+  const handleClick = () => {
+    // TODO
+  }
+
+  if (!response) return null
+
+  const partners = compact(response.hits)
+
+  return (
+    <>
+      <Text variant="lg" mb={4}>
+        {response.total ?? 0} Result{response.total === 1 ? "" : "s"}
+      </Text>
+
+      <GridColumns gridRowGap={4}>
+        {partners.map(partner => {
+          return (
+            <Column
+              key={partner.internalID}
+              span={[6, 4, 3]}
+              display="flex"
+              alignItems="flex-end"
+            >
+              <PartnerCellFragmentContainer partner={partner} mode="GRID" />
+            </Column>
+          )
+        })}
+      </GridColumns>
+
+      <Box textAlign="center" mt={4}>
+        <Button onClick={handleClick}>Show More</Button>
+      </Box>
+    </>
+  )
+}
+
+const PartnersFilteredCellsFragmentContainer = createFragmentContainer(
+  PartnersFilteredCells,
+  {
+    viewer: graphql`
+      fragment PartnersFilteredCells_viewer on Viewer
+        @argumentDefinitions(
+          category: { type: "[String]" }
+          near: { type: "String" }
+          page: { type: "Int", defaultValue: 1 }
+          type: { type: "[PartnerClassification]" }
+        ) {
+        filterPartners(
+          aggregations: [TOTAL]
+          defaultProfilePublic: true
+          eligibleForListing: true
+          near: $near
+          page: $page
+          partnerCategories: $category
+          size: 12
+          sort: RANDOM_SCORE_DESC
+          type: $type
+        ) {
+          total
+          aggregations {
+            counts {
+              name
+              value
+              count
+            }
+          }
+          hits {
+            internalID
+            ...PartnerCell_partner
+          }
+        }
+      }
+    `,
+  }
+)
+
+const PartnersFilteredCellsPlaceholder: FC = () => {
```

Comment:
Why formal component now? I've been following your lead and just rendering out as [plain JSX](https://github.com/artsy/force/blob/master/src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx#L87)

Reply:
Oh, I guess this one doesn't matter either way. Some of the others in this required a prop of `type`

Reply:
Ah i see 👍 Yeah was mostly just curious